lib/std/os/windows.zig

```diff
@@ -2859,12 +2839,66 @@ pub const LONG = i32;
 pub const ULONG64 = u64;
 pub const ULONGLONG = u64;
 pub const LONGLONG = i64;
-pub const HLOCAL = HANDLE;
-pub const LANGID = c_ushort;
+pub const LANGID = u16;
+
+/// pointer to immutable null-terminated u8 string
```

Comment:
> pointer

redundant

> immutable

redundant

> null-terminated

redundant

> u8

this is not an appropriate place to mention what `CHAR` is aliased to

> string

meaningless at best, misleading at worst